teamwork/templates/projects/update_post_update.html

```diff
@@ -0,0 +1,81 @@
+{% extends 'base.html' %}
```

Comment:
Looks good. We should add a message to display after editing/deleting an 'update'

Reply:
Done for Resources as well